code/game/objects/items/weapons/clown_items.dm

```diff
@@ -63,6 +74,11 @@
 		user.visible_message("[user] begins to scrub \the [target.name] out with [src].", "<span class='warning'>You begin to scrub \the [target.name] out with [src]...</span>")
 		if(do_after(user, src.cleanspeed, target = target))
 			to_chat(user, "<span class='notice'>You scrub \the [target.name] out.</span>")
+			if(istype(src, /obj/item/weapon/soap/vigilante) && istype(target, /obj/effect/decal/cleanable/crayon/gang))

```

Comment:
@Robustin ```istype(src), istype(src, some_path)``` will never be accepted, it is a CLEAR hint that you've done something wrong. Checking the type of src is a huge violation of OOP.